File: Java/retangulo/src/entities/Retangulo.java
package entities;

public class Retangulo {
	
	private double altura;
	private double largura;
	
	public Retangulo(double altura, double largura) {
		this.altura = altura;
		this.largura = largura;
	}
	
	public double calcularArea() {
		return altura * largura;
	}
	
	public double calcularDiagonal() {
		return Math.sqrt(Math.pow(altura, 2) + Math.pow(largura, 2));
	}
	
	public double calcularPerimetro() {
		return 2*(altura + largura);
	}
	
	@Override
	public String toString() {
		return "Area: " +String.format("%.2f", calcularArea()) + "\n"
	               + "Perímetro: " + String.format("%.2f", calcularPerimetro()) + "\n"
	               + "Diagonal: " + String.format("%.2f", calcularDiagonal()) + "\n";
	}
}

File: PHP/retangulo/src/entities/Retangulo.php
<?php

namespace Deivz\Retangulo\entities;

class Retangulo
{
    private float $altura;
    private float $largura;

    public function __construct(float $altura, float $largura)
    {
        $this->altura = $altura;
        $this->largura = $largura;
    }

    public function __get($atributo)
    {
        return $this->$atributo;
    }

    public function calcularArea(): float
    {
        return number_format($this->altura * $this->largura, 2);
    }

    public function calcularDiagonal(): float
    {
        return number_format(sqrt(pow($this->altura, 2) + pow($this->largura, 2)), 2);
    }

    public function calcularPerimetro(): float
    {
        return number_format(2*($this->altura + $this->largura), 2);
    }

    public function __toString()
    {
        return "Area: " . $this->calcularArea() . PHP_EOL
               . "Perímetro: " . $this->calcularPerimetro() . PHP_EOL
               . "Diagonal: " . $this->calcularDiagonal() . PHP_EOL;
    }
}